plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchSmokeTest.java

```diff
@@ -283,6 +285,173 @@ public void testArrayFields()
                 "VALUES ('hello', 20, 30, 60, NULL)");
     }
 
+    @Test
+    public void testAsRawJson()
+            throws IOException
+    {
+        String indexName = "raw_json";
+
+        @Language("JSON")
+        String mapping = "" +
```

Comment:
can we also add tests for cases for different combinations when `isArray` and `asRawJson` are true at different levels of nesting? If asRawJson is true at top level, the meta properties underneath will be inconsequential. I think that's okay, but would be good to test+document as well. Also, it'll be good to add tests for other data types.